src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -29,6 +29,7 @@
 
     private Collection<Pattern> tagsToExcludeFromChart = new ArrayList<>();
     private SortingMethod sortingMethod = SortingMethod.NATURAL;
+    private List<String> propertyFiles;
```

Comment:
also would like to have `classificationFiles` here so it tells me what it is used for and not how it is implemented

Reply:
updated PR will fix this